src/acceptance-test/java/net/consensys/orion/acceptance/send/receive/DualNodesSendReceiveTest.java

```diff
@@ -116,10 +129,17 @@ void setUpDualNodes(@TempDirectory final Path tempDir) throws Exception {
     secondHttpClient = vertx.createHttpClient();
     final Box.PublicKey pk1 = Box.PublicKey.fromBytes(decodeBytes(PK_1_B_64));
     final Box.PublicKey pk2 = Box.PublicKey.fromBytes(decodeBytes(PK_2_B_64));
+    final Box.PublicKey pk3 = Box.PublicKey.fromBytes(decodeBytes(PK_3_B_64));
+    final Box.PublicKey pk4 = Box.PublicKey.fromBytes(decodeBytes(PK_4_B_64));
+    final Box.PublicKey pk5 = Box.PublicKey.fromBytes(decodeBytes(PK_5_B_64));
     networkNodes = new ConcurrentNetworkNodes(NodeUtils.url("127.0.0.1", firstOrionLauncher.nodePort()));
 
-    networkNodes.addNode(pk1, NodeUtils.url("127.0.0.1", firstOrionLauncher.nodePort()));
-    networkNodes.addNode(pk2, NodeUtils.url("127.0.0.1", secondOrionLauncher.nodePort()));
+    networkNodes.addNode(
+        Arrays.asList(new Box.PublicKey[] {pk1, pk2}),
+        NodeUtils.url("127.0.0.1", firstOrionLauncher.nodePort()));
+    networkNodes.addNode(
+        Arrays.asList(new Box.PublicKey[] {pk3, pk4, pk5}),
```

Comment:
Can be simplified to  just: Arrays.asList(pk3, pk4. pk5)

Reply:
done